test/conformance/fixtures/python/test_driver.py

```
@@ -0,0 +1,210 @@
+import json
+import subprocess
+import sys
+from pathlib import Path
+
+
+DRIVER = Path(__file__).with_name("driver.py")
+
+
+def run_driver(*args: str) -> dict[str, object]:
+    completed = subprocess.run(
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)